.github/workflows/build.yml

```diff
@@ -58,7 +58,7 @@ jobs:
 
             - name: Start the local node
               if: success()
-              run: npx @hashgraph/hedera-local start -d --network local
+              run: npx @hashgraph/hedera-local start -d --network local --network-tag=0.50.0-alpha.0
```

Comment:
Should this be against a specific network tag or just against hedera-local@latest?